### PR TITLE
Arduino Cloud rename

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- As following the rename of the Arduino Cloud,I've updated the names across the tutorials.

Please keep an extra eye on the **changes in the comments in code**.

@karlsoderby @jacobhylen @BenjaminDannegard @Hannes7eicher please review the maker products and other tutorials
@jcarolinares @TaddyHC @canchebagur @mcmchris please review the pro products 

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by canchebagur — APPROVED** [review]
LGTM.

**Review by karlsoderby — APPROVED** [review]
LGTM!

File: content/hardware/04.pro/boards/portenta-c33/datasheets/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-c33/datasheets/datasheet.md:1]: checked

File: content/hardware/04.pro/boards/portenta-c33/tutorials/energy-meter-application-note/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-c33/tutorials/energy-meter-application-note/content.md:1]: checked

File: content/hardware/04.pro/boards/portenta-c33/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-c33/tutorials/user-manual/content.md:1]: checked

File: content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md:1]: checked

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:1]: "Cloud" as proper name, got it

checked

File: content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/portenta-x8-fundamentals.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/portenta-x8-fundamentals.md:1]: Checked

File: content/hardware/04.pro/boards/portenta-x8/tutorials/04.python-arduino-data-exchange/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/04.python-arduino-data-exchange/content.md:1]: checked

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md:1]: checked

File: content/hardware/04.pro/carriers/portenta-hat-carrier/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-hat-carrier/datasheet/datasheet.md:1]: checked

File: content/hardware/04.pro/carriers/portenta-max-carrier/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-max-carrier/datasheet/datasheet.md:1]: checked

File: content/hardware/04.pro/shields/portenta-vision-shield/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-vision-shield/datasheet/datasheet.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev1/datasheet.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev1/datasheet.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev2/datasheet.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev2/datasheet.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md:1]: not sure to be hones if we should always use "cloud" a proper name and therefore caps but is not critical and more than reasonable so ok!

checked1

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system/content.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system/content.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/features.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/features.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-sense-me/tutorials/connecting-to-iot-cloud/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-sense-me/tutorials/connecting-to-iot-cloud/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-sense-me/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-sense-me/tutorials/user-manual/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-vision/tutorials/image-classification/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-vision/tutorials/image-classification/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-vision/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-vision/tutorials/user-manual/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md
Comment (jhansson-ard) [reply content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md:1]: At least in places where we talk about "the Cloud", we should see it as a proper noun - similar to "the Internet". However, in words such as "cloud computing" or "cloud compatible", we don't **need** to capitalize the word. 

There are some discrepancies when searching for use-cases, so it is up to us to decide the use of it.

File: content/hardware/07.opta/opta-family/opta/_unlisted/getting-started/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/_unlisted/getting-started/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/datasheet/datasheet.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/energy-management-application-note/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/energy-management-application-note/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/home-automation-application-note/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/home-automation-application-note/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/opta-azure-iot-tutorial-en/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/opta-azure-iot-tutorial-en/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/tank-level-app-note/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/tank-level-app-note/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/user-manual/content.md:1]: checked

**Review by jcarolinares — COMMENTED** [review]
Pro Content changes reviewed after merge by the Pro Content Responsible.

Everything should be ok.

Thank you all for the changes.

**Pull request merged**